osu.Game/Overlays/KeyBinding/KeyBindingRow.cs

```diff
@@ -173,7 +178,8 @@ protected override bool OnMouseDown(MouseDownEvent e)
                 }
             }
 
-            bindTarget.UpdateKeyCombination(KeyCombination.FromInputState(e.CurrentState));
+            bindTarget.UpdateKeyCombination(keyCombinationSinceFocused(e.CurrentState));
```

Comment:
Even though it's one line, I'd probably make a method out of this usage pattern myself, as it's not immediately obvious `keyCombinationSinceFocused` should be used in all places.

Reply:
already have a method, might as well have this cover this line as well.